### PR TITLE
Add docs for Hex Cube Orange and Cube Yellow

### PR DESCRIPTION
**Pull request by hamishwillee**
This adds flight controller docs for Cube Orange and Yellow, and updates Cube Black. It also cross links from these to the quickstart.

The main point is not to fully document these - the Cube docs have now improved significantly. The purpose is to:that:
- highlight that these these targets are supported by PX4, and that they are manufacturer supported
- show how to build the firmware
- point through to the official cube docs.
- Highlight things they don't show so clearly - like serial port mappings.

I have assumed that the cube yellow and orange differ only in the processor.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@proficnc Can you sanity check this. I have used images and information from your store.
- Is it correct that the cube yellow and orange differ only in the processor?

File: en/flight_controller/cubepilot_cube_orange.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+The controller is designed to be used with a domain-specific carrier board in order to reduce the wiring, improve reliability, and ease of assembly.
+For example, a carrier board for a commercial inspection vehicle might include connections for a companion computer, while a carrier board for a racer could includes ESCs for the frame of the vehicle.
+
+Cube includes vibration isolation on two of the IMU's, with a third fixed IMU as a reference / backup.
+
+> **Note** This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+
+## Key Features
+
+* 32bit STM32F427VIT6 (32bit [ARM Cortex M7](https://en.wikipedia.org/wiki/ARM_Cortex-M#Cortex-M7), 400 MHz, Flash 2MB, RAM 1MB).
```
Comment (bkueng) [inline en/flight_controller/cubepilot_cube_orange.md:18]: STM32H743XIH6 (I think)

File: en/flight_controller/cubepilot_cube_yellow.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+The controller is designed to be used with a domain-specific carrier board in order to reduce the wiring, improve reliability, and ease of assembly.
+For example, a carrier board for a commercial inspection vehicle might include connections for a companion computer, while a carrier board for a racer could includes ESCs for the frame of the vehicle.
+
+Cube includes vibration isolation on two of the IMU's, with a third fixed IMU as a reference / backup.
+
+> **Note** This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+
+## Key Features
+
+* 32bit STM32F777 (32bit [ARM Cortex M7](https://en.wikipedia.org/wiki/ARM_Cortex-M#Cortex-M7), 400 MHz, Flash 2MB, RAM 512 KB).
```
Comment (bkueng) [inline en/flight_controller/cubepilot_cube_yellow.md:18]: From the board config it's an STM32F765II

Comment (dagar) [reply en/flight_controller/cubepilot_cube_yellow.md:18]: Pretty sure that's a bug.

Comment (dagar) [reply en/flight_controller/cubepilot_cube_yellow.md:18]: https://github.com/PX4/Firmware/pull/15414

File: en/flight_controller/cubepilot_cube_orange.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_controller/cubepilot_cube_orange.md]: Thanks. I modified to STM32H743ZI which is what the defconfig said.

**Comment by hamishwillee** [comment]
Thanks. I modified both processors to match the current defconfig - e.g. https://github.com/PX4/Firmware/blob/master/boards/cubepilot/cubeorange/nuttx-config/nsh/defconfig#L30

I'll merge now and we can fix anything else as discovered.

**Pull request merged**